README.md

```diff
@@ -814,3 +814,7 @@ Bottlerocket operates with two default storage volumes.
 * The root device, `/dev/xvda`, holds the active and passive [partition sets](#updates-1).
   It also contains the bootloader, the dm-verity hash tree for verifying the [immutable root filesystem](SECURITY_FEATURES.md#immutable-rootfs-backed-by-dm-verity), and the data store for the Bottlerocket API.
 * The data device, `/dev/xvdb`, is used as persistent storage for container images, container orchestration, [host-containers](#Custom-host-containers), and [bootstrap containers](#Bootstrap-containers-settings).
+
+Bottlerocket will increase the partition size to use all of `/dev/xcdb` on boot.
+If you increase the size of that device, you can reboot Bottlerocket to extend the data partition.
```

Comment:
It's not guaranteed that the device will be `xvdb`:
* on Nitro instance types, it will be `nvme1n1`
* on variants using "unified", it could be `xvda` or `nvme0n1`

However, I'm not sure there's a great way to introduce that level of detail here; we'd need to rewrite it a bit first to mention "split" vs "unified". It might help to just remove the specific device names.